src/vstart.sh

```diff
@@ -117,7 +117,7 @@ fi
 [ -z "$CEPH_NUM_MON" ] && CEPH_NUM_MON=3
 [ -z "$CEPH_NUM_OSD" ] && CEPH_NUM_OSD=3
 [ -z "$CEPH_NUM_MDS" ] && CEPH_NUM_MDS=3
-[ -z "$CEPH_NUM_MGR" ] && CEPH_NUM_MGR=1
+[ -z "$CEPH_NUM_MGR" ] && CEPH_NUM_MGR=
```

Comment:
Careful, don't actually commit this.

Reply:
sure.  I dint realize these changes got pulled too. Will update.